agot-bg-game-server/src/common/ingame-game-state/game-data-structure/house-card/AeronDamphairDwDHouseCardAbility.ts

```diff
@@ -1,12 +1,21 @@
 import HouseCardAbility from "./HouseCardAbility";
 import HouseCard from "./HouseCard";
 import House from "../House";
+import CombatGameState from "../../action-game-state/resolve-march-order-game-state/combat-game-state/CombatGameState";
 import BeforeCombatHouseCardAbilitiesGameState from "../../action-game-state/resolve-march-order-game-state/combat-game-state/before-combat-house-card-abilities-game-state/BeforeCombatHouseCardAbilitiesGameState";
 import AeronDamphairDwDAbilityGameState from "../../action-game-state/resolve-march-order-game-state/combat-game-state/before-combat-house-card-abilities-game-state/aeron-damphair-dwd-ability-game-state/AeronDamphairDwDAbilityGameState";
 
 export default class AeronDamphairDwDHouseCardAbility extends HouseCardAbility {
+    abilityGameState: AeronDamphairDwDAbilityGameState | null = null;
```

Comment:
I like this idea! Let's integrate this to the base class for all abilities which may require a pre state before modifying combat strength!